tests/test_rate_limiter.py

```diff
@@ -60,23 +129,129 @@ def test_rate_limiter_with_redis(fake_all_blocks, limiter_test):
         resp = response.json()
         assert all(b['type'] != "wikipedia" for b in resp['blocks'])
 
-def test_rate_limiter_without_redis(fake_all_blocks):
+def test_rate_limiter_without_redis(louvre_museum):
     """
     Test that Idunn doesn't stop external requests when
     no redis has been set: 10 requests to Idunn should
     generate 10 requests to Wikipedia API
     """
     client = TestClient(app)
-    with responses.RequestsMock(assert_all_requests_are_fired=False) as rsps:
+
+    WikipediaLimiter._limiter = None
+    settings._settings['WIKI_API_REDIS_URL'] = None
+    with responses.RequestsMock() as rsps:
         rsps.add('GET',
              re.compile('^https://.*\.wikipedia.org/'),
              status=200,
              json={"test": "test"}
         )
-
         for i in range(10):
             response = client.get(
-               url=f'http://localhost/v1/pois/{fake_all_blocks}?lang=es',
+               url=f'http://localhost/v1/pois/{louvre_museum}?lang=es',
             )
 
         assert len(rsps.calls) == 10
+
+
+def restart_wiki_redis(docker_services):
+    """
+    Because docker services ports are
+    dynamically chosen when the service starts
+    we have to get the new port of the service.
```

Comment:
is'nt this beecause we do not have stopped the previous service ?

Reply:
Wasn't it stop line 211 ?